content/rest-api/teams.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Teams API
+description: API for managing team members
+
+weight: 5
+---
+
+This document describes the API endpoints for managing teams. To use this API, you must be a **team owner**.
+
+
+>Read more about the [Teams](../getting-started/teams/) feature, the available user roles and permissions.
+
+
+
+## Invite a new team member
+Invite a new team member to your team.
+
+`POST /:team_id/invitation`
+
+#### Parameters
+
+
+| **Name** | **Type** | **Description**                                            |
+|----------| -------- |------------------------------------------------------------|
+| `email`  | `string` | **Required.** SSH or HTTPS URL for cloning the repository. |
+| `role`   | `string` | **Required.** Could be `owner` or `developer`              |
+
+#### Example
+
+{{< highlight bash "style=paraiso-dark">}}
+curl -H "Content-Type: application/json" \
+     -H "x-auth-token: <API Token>" \
+     -d '{
+        "email": "your_user@domain.com",
+        "role": "developer"
+     }' \
+     -X POST https://api.codemagic.io/<team_id>/invitation
+{{< /highlight >}}
+
+
+#### Response
+
+A full team object is returned. You can see that your invitation is created
+in the `invitations` section: 
+
+
+{{< highlight json "style=paraiso-dark">}}
+{
+  "team":{
+    "_id":"<team_id>",
+    "applicationIds":[...],
+    "billing":{...},
+    "buildTimes": {...},
+    "collaborators":[...],
+    "invitations": [
+        {
+            "invitedFrom":"email",
+            "isPendingRemoved":false,
+            "role":"developer",
+            "addedBy":"<user_id>",
+            "updatedAt": "<update_date>",
+            "createdAt": "<create_date>",
+            "email":"your_user@domain.com",
+            "permissions": [...],
+            "inviteCode":"<invite_code>"}
+        ],
+        ...
+        ...
+    }
+}
+{{< /highlight >}}
+
+
+## Change a team member role
+
+
+Change the role of your team members. The available roles are `owner` and `developer` (corresponds to "member" in the UI).
+
+`PUT /:team_id/collaborator/:user_id`
+
+#### Parameters
+
+
+| **Name** | **Type** | **Description**                                            |
+|----------| -------- |------------------------------------------------------------|
+| `role`   | `string` | **Required.** Could be `owner` or `developer`              |

```

Comment:
is there a way to find `user id` from previous response?

Reply:
Yes, you can find it in `collaborators` section, but for me not make sense to have to create an invitation just to get this information. That's why I suggest in PR description to add the GET endpoint.